### DISCUSSION TITLE
Any way to supress warning on import?

### DISCUSSION THREAD

**Discussion by Alex-Merrill** [colour-science/colour#1237] [Q&A]
Hi there,

The following warning is given immediately following importing colour:

```
ColourUsageWarning: "Matplotlib" related API features are not available: "No module named 'matplotlib'".
See the installation guide for more information: https://www.colour-science.org/installation-guide/
```

I would like to disable this warning, but can't seem to figure out how that would be done. The colour.utilities.supress_warnings method of course can't be called before importing the package, is there some other way to do this? I would prefer to not install matplotlib as I am not using it in the project.

Thanks!

**Comment by thomasmansencal** [comment]
Hi @Alex-Merrill,

Sorry, there is no way to do that but I made a note a while ago to be able to do this, at least with an environment variable.

<img width="976" alt="image" src="https://github.com/colour-science/colour/assets/99779/a9867b2a-0865-4b61-b508-56e8de86d91c">

Cheers,

Thomas

**Reply by Alex-Merrill** [reply]
Thanks for the reply @KelSolaar. What do you mean by this- "but I made a not a while ago to be able to do this"? An issue or PR? I don't see any issues or PRs related to this, but I might be missing something.

**Reply by thomasmansencal** [reply]
"I made a not**e**"

**Reply by thomasmansencal** [reply]
I implemented a way to do that in #1239.
